arch/arm/soc/st_stm32/stm32l4/dts.fixup

```diff
@@ -62,6 +62,12 @@
 #define CONFIG_I2C_4_ERROR_IRQ                  ST_STM32_I2C_V2_40008400_IRQ_ERROR
 #define CONFIG_I2C_4_BITRATE                    ST_STM32_I2C_V2_40008400_CLOCK_FREQUENCY
 
+#define CONFIG_RTC_0_BASE_ADDRESS               ST_STM32_RTC_40002800_BASE_ADDRESS
+#define CONFIG_RTC_0_IRQ_PRI                    ST_STM32_RTC_40002800_IRQ_0_PRIORITY
+#define CONFIG_RTC_0_IRQ                        ST_STM32_RTC_40002800_IRQ_0
+#define CONFIG_RTC_0_NAME                       ST_STM32_RTC_40002800_LABEL
+#define CONFIG_RTC_PRESCALER                    ST_STM32_RTC_40002800_PRESCALER
```

Comment:
Where is that used?

Reply:
Only `...BASE_ADDRESS` isn't used anywhere, just there for completeness. The address is used as defined by the ll drivers, simply `RTC`. `CONFIG_RTC_PRESCALER` is used in `include/rtc.h` to scale `RTC_ALARM_SECOND`.

The rest are used by the `INIT` macro and `irq_config()`.

Reply:
Ok. Not sure it is the right approach, but since used already in NXP driver, let's keep it that way.